#31 JavaDoc for MusicPlayer

nfetissow opened the pull request:
JavaDoc comments for MusicPlayer class

nfetissow committed:
#31 JavaDoc for MusicPlayer

JavaDoc comments for MusicPlayer class

xammi commented on app/src/main/java/ru/bmstu/wundermusik/MusicPlayer/MusicPlayer.java:
```diff
@@ -22,6 +22,13 @@
 import ru.bmstu.wundermusik.models.Track;
 
+/***
+ * Этот класс представляет собой движок плеера.
```
Старайся избегать лишних слов: "Движок плеера." вполне достаточно. Понятно же, что комментарий относится к этому классу

xammi commented on app/src/main/java/ru/bmstu/wundermusik/MusicPlayer/MusicPlayer.java:
```diff
@@ -42,6 +49,12 @@
 
+    /***
+     * Обработчик событий, пришедших по EventBus
+     * В зависимости от типа входного параметра вызывается одна из реализаций
```
непонятно, "вызывается одна из реализаций" чего?

nfetissow committed:
#31 Corrections

Removed triple asterisks;
Removed redundant words;
Changed comment for OnEvent methods

xammi commented:
названия пакетов пишем нижним регистром, не MusicPlayer/MusicPlayer.java

nfetissow committed:
#31 changed name of package, javaDoc for Events

deprecated TrackEndedEvent - Eugene will take care of that

xammi commented:
Мерджим.
